src/jsonSass.js

```diff
@@ -22,5 +22,6 @@ function jsonSass(options) {
 }
 
 module.exports = jsonSass;
+module.exports.convertJs = jsToSassString;
 export default jsonSass;
 export { jsToSassString as convertJs };
```

Comment:
Looks like it's an issue with how babel exports things with only a default or with a default and other properties: http://babeljs.io/docs/usage/modules/#interop
(Or not quite an issue, it's by design). Anyway, I have a different fix for it in #9.